demo/Meter.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<title>Meter Styling</title>
+		<link rel="stylesheet" href="../dist/css/chassis.css">
+	</head>
+	<body>
```

Comment:
Per jQuery HTML Style Guide, `<body>` and `<html>` tags shouldn't be indented: http://contribute.jquery.org/style-guide/html/#indentation